aibolit/patterns/many_primary_ctors/many_primary_ctors.py

```diff
@@ -0,0 +1,43 @@
+import javalang.ast
+import javalang.parse
+import javalang.tree
+
+
+class ManyPrimaryCtors(object):
+    def value(self, filename: str):
```

Comment:
@tribals Why not adding return type `-> []` to the method as you do in other methods ?

Reply:
@tribals I think a better name for the class would be `MultiPrimaryCtors`. 
And the file name as well.

Reply:
I think this is just wording. I named pattern (and class) according to [issue](#99)

Reply:
> @tribals Why not adding return type `-> []` to the method as you do in other methods ?

A `[]` return type is just not valid return type. It must be `typing.List[int]`. Also, it is not just useful to specify typing if you actually didn't bother with it's checking using third-party tools (like [mypy](http://mypy-lang.org/)).

Reply:
@tribals I find it a little bit difficult to review, when I cannot see the return types.

Reply:
So, you should expect it, because Python is dynamically typed language. Type hints are completely optional.

Reply:
@tribals Alright